src/main/java/com/example/bookwonders/dto/cart/AddCartItemRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package com.example.bookwonders.dto.cart;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class AddCartItemRequestDto {
+    @NotBlank
```

Comment:
```suggestion
    @NotNull
```
@NotBlank Annotation on Numeric Fields: The @NotBlank annotation is used to check that a character sequence's trimmed length is not empty, so it's inappropriate for Long or int types. It's usually used for String fields.